### PR TITLE
feat: Add author to log when a project is accepted or denied

### LINKED ISSUE #43: Feature request: Add AuthorID to the approval/rejection log

### PR DESCRIPTION
**Pull request by nullishamy**
This adds the project author to the log when a project is accepted or
denied. This also makes the project ID part of the log clearer by adding
a label. Closes #43.

### CODE REVIEW

File: src/utils/processReaction.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default async (discordData: ShowcaseDiscordData, internalData: ShowcaseDa
     const staffVotes = { up: project.upvotes.staff, down: project.downvotes.staff }
     const veteranVotes = { up: project.upvotes.veterans, down: project.downvotes.veterans }
 
-    log.info(`Project ${project.name} (${project.id}) was ${wasApproved ? 'approved' : 'rejected'} with ${staffVotes.up + veteranVotes.up} upvotes [Staff/vet spread: ${staffVotes.up} | ${veteranVotes.up}] and ${staffVotes.down + veteranVotes.down} downvotes [Staff/vet spread: ${staffVotes.down} | ${veteranVotes.down}]`)
+    log.info(`Project ${project.name} (ID: ${project.id}) (Author: <@!${project.author}>, ${project.author}) ${wasApproved ? 'approved' : 'rejected'} with ${staffVotes.up + veteranVotes.up} upvotes [Staff/vet spread: ${staffVotes.up} | ${veteranVotes.up}] and ${staffVotes.down + veteranVotes.down} downvotes [Staff/vet spread: ${staffVotes.down} | ${veteranVotes.down}]`)
```
Comment (jtsshieh) [inline src/utils/processReaction.ts:44]: Maybe change the format to (ID: ..., Author: ...)

Comment (jtsshieh) [reply src/utils/processReaction.ts:44]: Otherwise lgtm

**Review by jtsshieh — CHANGES_REQUESTED** [review]

File: src/utils/processReaction.ts (hunk 1 above)
Comment (nullishamy) [reply src/utils/processReaction.ts]: Done

**Comment by nullishamy** [comment]
Also forgot to change the `safeSendMessage` string too, so that was updated.

**Review by jtsshieh — APPROVED** [review]

**Comment by nullishamy** [comment]
Should we resolve the ID to a name? That would put it in line with other logs (in this specific project but also TCD as a whole). The ID should always resolve as we will be checking this beforehand.

**Comment by nullishamy** [comment]
Also updated the allowed mentions option to disable pings in logs.

**Review by BobobUnicorn — APPROVED** [review]

**Pull request merged**